winml/lib/Api.Core/WinMLAdapter.cpp

```diff
@@ -57,8 +57,8 @@ class AbiSafeTensor : public Microsoft::WRL::RuntimeClass<
                           Microsoft::WRL::RuntimeClassFlags<Microsoft::WRL::ClassicCom>,
                           ITensor> {
  private:
-  onnxruntime::Tensor& tensor_;  // weak ref
-  ComPtr<IOrtValue> value_;      // strong ref
+  onnxruntime::Tensor& tensor_;              // weak ref
+  Microsoft::WRL::ComPtr<IOrtValue> value_;  // strong ref
```

Comment:
>Microsoft::WRL:: [](start = 2, length = 16)

Why is explicit namespace needed?

Reply:
Oh because winrt also has com ptrs so it makes it more clear 

---
In reply to: [350335004](https://github.com/microsoft/onnxruntime/pull/2469#discussion_r350335004) [](ancestors = 350335004)